styleguide/derek/_partials/_footer.html

```diff
@@ -0,0 +1,5 @@
+<!--[footernav]-->
+<nav class="">
```

Comment:
Sure thing. Do you want to do the same for the ceiling and main nav @bighappyface ?